packages/mobile/src/redux/migrations.ts

```diff
@@ -121,7 +120,36 @@ export const migrations = {
     if (!lastUsedProvider || !lastUsedProvider.name) {
       return state
     }
-    const lastProvider = Object.entries(providersDisplayInfo).find(
+
+    const providerDisplayInfo = {
```

Comment:
Including this as a `const` because I deleted the import. I wasn't sure what the best practice was with dealing with outdated migrations that import data that no longer exists

Reply:
We can probably remove this and just set the `lastUsedProvider` as `null` to make sure we don't break anything.

Reply:
I'm worried that we will lose our ability to back test certain migrations. I don't know why we'd want to do this but for example if we wanted to test the upgrade experience from v7 to v8. It makes me a little nervous to not have any record of what we were doing in the past with migrations.

Perhaps the better principle is no imports into migrations? It's a little annoying to redefine thing but it may be helpful in the future to have an unchanged record.